react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -26,17 +27,21 @@ export default class AutocompleteInput extends Component {
   }
 
   render() {
-    const { onGoToSearchPage, onClearInput, compactMode, value, ...restProps } = this.props
+    const { onGoToSearchPage, onClearInput, compactMode, value, hasIconLeft, iconClasses, ...restProps } = this.props
 
     const suffix = (
       <span className="flex items-center pointer" onClick={value ? onClearInput : undefined} >
```

Comment:
This `onClick` shouldn't just be `onClick={value && onClearInput}`?

Reply:
Yes, it should.